### ISSUE TITLE
Server crash loop 2-3 minutes after booting 0.11.917-ALPHA.

### ISSUE THREAD

**Issue by patgoull** [ldtteam/minecolonies#4868]
### Minecolonies version
Version:  0.11.917-ALPHA

### Actual behaviour
After updating, we played around 2 hours before getting a server crash. After booting it up again, we keep getting the crash after 2-3 minutes. We had issues and with builder having trouble building and requesting items.

### Logs: Latest.log and crashlog at https://gist.github.com/
https://gist.github.com/patgoull/e6199f49a4c69f7a9f7611a8651cdf30

**Comment by Raycoms** [comment]
Can you send me this world?

**Comment by patgoull** [comment]
Here is the world zipped : https://drive.google.com/open?id=1lfLBNz58D18rR6KhhzAvrB5StZz32SK0

Pretty sure the colony that cause the issue is ID 3. Logs are mentionning it's builder. Ask if you need any additonnal infos.

**Comment by PatrickGoulet** [comment]
I had builders exceptions earlier but this latest.log is about dman. My bad for the confusion.

**Comment by Raycoms** [comment]
Can you try out the newest?

**Comment by patgoull** [comment]
Still got the problem on Version: 0.11.924-ALPHA. Same world should do to reproduce.

https://gist.github.com/patgoull/9267f23c52d82d54a873af05ed381f35

**Comment by patgoull** [comment]
I updated to alpha-928 and tried to make my builder do the university again (clearly, in my case, this is the building causing the issue when it's in progress of beeing built even tho it's probably not building specific). The server was stable a while but when it was almost done I crashed again with this log : https://gist.github.com/patgoull/6012802999cf81b7e6693035a73e7e6f

The rest of the colony is fine. I did 2-3 new building without issues, RS is working fine and all.

Here is the world with 928 : https://drive.google.com/open?id=1lfLBNz58D18rR6KhhzAvrB5StZz32SK0

Just in case it's related, my wharehouse racks are having the issue where they conflict with the ones next to them depending and how to interract with them. They switch orientations and cna be invisible until you interact with them

**Comment by PatrickGoulet** [comment]
Anything on this @Raycoms ? Is my world beyond repair hahaha. I feel like the previous fix worked for the majority of people so I am not sure what to do on this. I might just place the university in creative mode and keep on playing.

**Comment by Raycoms** [comment]
I can try out your world in a bit =)

**Comment by PatrickGoulet** [comment]
If you do so, I forgot to mention the crash occur in the colony ID 2. Just loading it would probably resume the error quite fast.

**Comment by Raycoms** [comment]
Okay, I'll make a PR for the mechanic now, with the content of that PR I cannot reproduce a crash in your world. and I just spent an hour in your world

**Comment by PatrickGoulet** [comment]
Im gonna test the world as soon as PR 4933 is out in a release. Thanks for the awesome support.

**Comment by patgoull** [comment]
After updating and playing for around 10h on 2 colonies, I got another ticking entity crash on 937-ALPHA. During the 10h, we had some AI issues on builders and DMan but nothing crazy. We also noticed RS was not detecting many items in the warehouse even tho the inventory UI was detecting it. At least the Dman were not able to go pick it up.

https://gist.github.com/patgoull/2ced3af1e371a87a057dd960f8a24640

Might not be related but we were investigating why a farmer had not harvest for 30 min and we were messing with him. I tried a restart on him and it crashed 10 sec after. Mentionning it just in case cause the timing was odd.

EDIT : I just noticed your PR for tools fufillment and request update. Im going to test it when it's out and tell you if it happens again.
